apps/condo/domains/miniapp/components/AppDescription/AboutB2BAppPage.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useEffect } from 'react'
+import { useRouter } from 'next/router'
+import get from 'lodash/get'
+import Error from 'next/error'
+import Head from 'next/head'
+import { useOrganization } from '@core/next/organization'
+import { useIntl } from '@core/next/intl'
+import { B2BApp, B2BAppContext } from '@condo/domains/miniapp/utils/clientSchema'
+import { B2B_APP_TYPE } from '@condo/domains/miniapp/constants'
+import LoadingOrErrorPage from '@condo/domains/common/components/containers/LoadingOrErrorPage'
+import { PageWrapper, PageContent } from '@condo/domains/common/components/containers/BaseLayout'
+import { AppDescriptionPageContent } from './AppDescriptionPageContent'
+
+
+
+interface AboutB2BAppPageProps {
+    id: string,
+}
+
+
+export const AboutB2BAppPage: React.FC<AboutB2BAppPageProps> = ({ id }) => {
+    const intl = useIntl()
+    const LoadingMessage = intl.formatMessage({ id: 'Loading' })
+    const router = useRouter()
+    const userOrganization = useOrganization()
+    const organizationId = get(userOrganization, ['organization', 'id'], null)
+
+    const { obj: app, error: appError, loading: appLoading } = B2BApp.useObject({
+        where: { id },
+    })
+    const appId = get(app, 'id', null)
+    const { obj: context, error: contextError, loading: contextLoading } = B2BAppContext.useObject({
+        where: {
+            app: { id: appId },
+            organization: { id: organizationId },
+        },
+    })
+
+    const redirectUrl = `/miniapps/${id}?type=${B2B_APP_TYPE}`
```

Comment:
No `id` validation before creating url

Reply:
Also fixed in parent component:
https://github.com/open-condo-software/condo/pull/1540/commits/18f0289d907d2927cc33752419123b6b9a4e0d46#diff-df367fbd634530321654d967e46e900918718f3cd3cf751c4bddcc27e681d01a